test_shopcart.py

```diff
@@ -29,6 +29,175 @@ def test_index(self):
         self.assertTrue ('Shopcart Demo REST API Service' in resp.data)
 
 
+    def test_create_shopcarts(self):
```

Comment:
The reason I think it should be divided into small test function is that when a test fails, it would be much easier to figure out what test failed.